digits/dataset/images/classification/forms.py

```diff
@@ -21,13 +21,16 @@ class ImageClassificationDatasetForm(ImageDatasetForm):
                                   choices=[
                                       ('lmdb', 'LMDB'),
                                       ('hdf5', 'HDF5'),
+                                      ('tfrecords', 'TFRecords'),
```

Comment:
Hey, come on, it seems you've adopted nothing from our previous reviews. This should be removed, so that we do have the back-end implementation for it, but people don't use it as it's currently not working as it should.